popup/pages/send.tsx

```diff
@@ -0,0 +1,141 @@
+import * as ethers from "ethers"
+import {
+  useCallback,
+  useContext,
+  useEffect,
+  useState,
+  type ChangeEvent
+} from "react"
+import { useLocation } from "wouter"
+
+import { ProviderCtx } from "~popup/ctx/provider"
+import { PopupPath } from "~popup/util/page"
+import type { BigNumberish, HexString } from "~typing"
+
+export function Send() {
+  const providerCtx = useContext(ProviderCtx)
+  const [_, setLocation] = useLocation()
+  const [txHash, setTxHash] = useState<HexString>("")
+
+  // Input state
+  const [txTo, setTxTo] = useState<HexString>("")
+  const [txToSuggestions, setTxToSuggestions] = useState<HexString[]>([""])
+  const [txValue, setTxValue] = useState<BigNumberish>("0")
+  const [toBorderColor, setToBorderColor] = useState<string>("border-gray-300")
+  const [amountBorderColor, setAmountBorderColor] =
+    useState<string>("border-gray-300")
```

Comment:
These are more about status. The logic actually is "when error occurred, we should show red border for inputs". 

So in state we should store page status, and render screen according to the status, instead of putting rendering details into state.